### DISCUSSION TITLE
Code list sign off on opencodelists

### DISCUSSION THREAD

**Discussion by lemanska** [opensafely/documentation#554] [Codelists] [answered]
Hi all and @inglesp please could you clarify on how the code list sign off happens. I am creating a code list with a colleague (ID 42ae96de). The code list was created by my colleague and I will be signing it off. When I access the codelist I have no option to approve it. Is it because I have no necessary permissions or is there another reason for this? My user name on the opencodelists.org is agleman. 
Thanks a lot!

**Comment by inglesp** [comment, marked as answer]
Hi @lemanska, this is a good question!  The workflow around codelist signoff is a bit half-arsed (to use the technical term) and at the moment it's only possible to add a signoff for a codelist that belongs to an organisation, and not to one that belongs to a user (as in your case).

I'm not sure when we'll get the chance to revisit this.  For now, if your colleague wants to record that you've signed off on their codelist, they could add a note to the codelist description.

**Reply by lemanska** [reply]
Thank you - that is all clear.

**Reply by SRW612** [reply]
I have also come up against the same issue. Just to clarify @inglesp, the codelist [documentation](https://github.com/opensafely/documentation/blob/main/docs/codelist-creation.md) outlines a workflow that includes signoff, but signoff is only possible from an organisational account?

Does this also mean that only codelists from an organisational account can become public, since this requires signoff by 2+ people?
